Autopilot/AttitudeManager/Src/MahonyAHRS.cpp

```diff
@@ -29,7 +29,7 @@
 
 volatile float twoKp = twoKpDef;											// 2 * proportional gain (Kp)
 volatile float twoKi = twoKiDef;											// 2 * integral gain (Ki)
-volatile float q0 = 0.52513f, q1 = 0.051342f, q2 = 0.014727f, q3 = 0.84934f;					// quaternion of sensor frame relative to auxiliary frame
+volatile float q0 = 1.0f, q1 = 0.0f, q2 = 0.0f, q3 = 0.0f;					// quaternion of sensor frame relative to auxiliary frame
```

Comment:
there a particular reason we are settnig these to 0?

Reply:
If the imu was attached crooked we could specify that here. This is set to the default of no rotation, so the imu is assumed to be lined up with the forward direction.